lms/templates/view.html

```diff
@@ -28,23 +29,34 @@ <h1>{{ _('תצוגת תרגיל') }} {{ solution['exercise']['id'] }}: {{ soluti
       {% endfor -%}
       </ul>
     </nav>
-    {% if is_manager %}
+    {% if config.SHAREABLE_SOLUTIONS %}
+    <div class={{ direction }}>
+      <strong>{{ _('שיתוף תרגיל') }}:</strong>
+    </div>
+    <div id="share-solution" class="share-button{%- if shareable_by_user %} shared{% endif -%}">
+      <span id="solution-link"><i class="fa fa-share-alt" title="Enable or disable solution share"></i></span>
+      <span id="get-shared-link"><i class="fa fa-cloud" title="Copy solution shared link"></i></span>
+      <span id="shared-link-text">Link copied</span>
+    </div>
+    {% endif -%}
+    {% if is_manager and not is_shared %}
         <div class="progress manager-progress-bar">
             <div class="progress-bar" style="width: {{ left }}%;" role="progressbar" aria-valuenow="{{ left }}" aria-valuemax="100">
                 {{ left }}% {{ _('לסיום בדיקה') }}
             </div>
         </div>
     {% endif %}
+    {%- endif %}
   </div>
   <div id="view-body">
     <div class="code-view-container">
-      <div id="code-view" data-id="{{ solution['id'] }}" data-file="{{ current_file.id }}" data-exercise="{{ solution['exercise']['id'] }}" data-role="{{ role }}">
+      <div id="code-view" data-id="{{ solution['id'] }}" data-file="{{ current_file.id }}" data-exercise="{{ solution['exercise']['id'] }}" data-role="{{ role }}" data-shared="{{ config.SHAREABLE_SOLUTIONS }}" data-user-shared="{{ shareable_by_user }}">
```

Comment:
Why would you need the data-shared attribute?

Reply:
I use it in the my.js file in the following lines:
139, 144 and 23.